testing/test_monkeypatch.py

```diff
@@ -328,7 +330,7 @@ def test_importerror(testdir):
     p = testdir.mkpydir("package")
     p.join("a.py").write(
         textwrap.dedent(
-            """\
+            r"""\
```

Comment:
Not sure why this was added, but it is the reason why the build broke. 😁

Reply:
I've added an issue on pyupgrade reporting the issue ;)
https://github.com/asottile/pyupgrade/issues/76